### PR TITLE
Use checksum annotation and increase wait timeout

### PR DESCRIPTION
**Pull request by rossf7**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/chart-operator/values.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ helm:
   http:
     clientTimeout: "5s"
   kubernetes:
-    waitTimeout: "10s"
+    waitTimeout: "120s"
```
Comment (rossf7) [inline helm/chart-operator/values.yaml:13]: 10s is very short for charts with hooks. So I think its better to use the same default we have in the catalog. WDYT?

**Review by tomahawk28 — APPROVED** [review]
LGTM, 👍

File: helm/chart-operator/values.yaml (hunk 1 above)
Comment (tomahawk28) [reply helm/chart-operator/values.yaml:13]: Sure, Let's go with 120s.

**Comment by rossf7** [comment]
Tested on `godsmack`.

**Pull request merged**